pkg/common/utils/sbom_utils/sbom.go

```diff
@@ -0,0 +1,29 @@
+package sbom_utils
```

Comment:
As per go convention, we should use a package name such as `sbomutils`

> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps

https://go.dev/doc/effective_go